vm/src/bench/mod.rs

```diff
@@ -74,8 +77,11 @@ where
         Ok(raw.map(|a| ActorState {
             code: a.code,
             state: a.state,
-            sequence: a.sequence,
+            call_seq: a.sequence,
             balance: a.balance,
+            // TODO: possibly rename predictable address if these are the same concept
+            // in ref-fvm predictable address is assigned to delegated address in some instances
```

Comment:
These are the same concept as far as I know.

Reply:
Will do in a followup: requires changing and landing in builtins-first